runtime.go

```diff
@@ -584,7 +584,7 @@ func getWorkingDir(basedir string, codeuri string, checkWorkingDirExist bool) (s
 		if _, err := os.Stat(dir); err != nil {
 			// It doesn't, so just use the directory of the SAM template
 			// which might have been passed as a relative directory
-			dir = basedir
+			dir = abs
```

Comment:
This was the cause of the bug. 
If the base directory was empty (e.g. if they called --template filename.yaml), and the CodeUri wasn't a valid relative directory that exists, then instead of returning the current working directory, it returned "".